src/runtime/test/artifacts/Profile/Profile.recipes

```diff
@@ -27,10 +26,3 @@ recipe BasicUserProfile
     people <- people
   AddressForm
     address = address
-
-recipe FavoriteFood
```

Comment:
should this be disappearing here too?

Reply:
I'm not sure what you mean. That line is in fact being deleted, so Profiles.recipes no longer has any mention of FavoriteFood. There is a FavoriteFood.schema, which is pretty much content-free and hence completely independent of the stuff I'm deleting, used by the Restaurant stuff, so it's staying for now.